### DISCUSSION TITLE
Artifact Attestations are awesome!

### DISCUSSION THREAD

**Discussion by jay0lee** [community/community#122193] [Actions]
### Select Topic Area

Product Feedback

### Body

I'm excited to see Artifact Attestations reach public beta? I've been researching ways to allow end users to validate the binary they are running is valid and ties back to my project's repo and build actions. Having this built into GitHub means I don't need to deal with PKIs, secrets and all the additional complexity. This is a big win for open source projects!

I do wish it was slightly easier for end users to validate their file came from my repo. Today it looks like they need to:
1. Install gh command line tool.
2. Run something like:
  ```gh attestation verify -o MY-ORG their-file.dat```

to validate that `their-file.dat` originated with my GitHub organization. That's good but many of my users don't necessarily use GitHub command line tools. In an ideal world they'd be able to run something like:
```
file_hash=$(sha256sum their-file.dat | cut -d' ' -f1)
curl "https://api.github.com/orgs/MY-ORG/attestations/sha256:${file_hash}"
```

and using standard *nix tools the user could verify the binary came from my project. However the API seems to require authentication and a URL like ```https://github.com/MY-ORG/MY-REPO/attestations/sha256:$file_hash``` doesn't actually work, the web page view seems to only work via the numeric ID of the attestation.

Is there some other method to anonymously take the sha256sum of a given file and search against an organization or repo with it?

**Comment by AnimMouse** [comment]
You can search the sha256sum of a file here: https://search.sigstore.dev
